assets/src/scripts/inc/history.js

```diff
@@ -0,0 +1,282 @@
+var React    = require( 'react' );
+var ReactDOM = require( 'react-dom' );
+var diff_match_patch = require("exports?diff_match_patch!./../vendor/diff_match_patch.js");
+
+/**
+ * Diff component.
+ */
+var PageHistoryDiff = React.createClass({
+
+	getDefaultProps: function() {
+	    return {
+	    	diff_a: null,
+	    	diff_b: null,
+	    }
+	},
+
+	render: function() {
+		return ( <div className="PageHistory_Diff" dangerouslySetInnerHTML={ { __html: this.getDiffHTML() } } /> );
```

Comment:
Rather than setting HTML, since you're building this dynamically, it'd be better to use `<div>{ this.getDiffHTML() }</div>` and return elements from `getDiffHTML()`